### ISSUE TITLE
'strip_common_prefix' removing too much

### ISSUE THREAD

**Issue by iBotPeaches** [getsentry/action-release#162]
### Environment

v1 

### Steps to Reproduce

```yaml
    - name: Create Sentry release
      uses: getsentry/action-release@v1
      env:
        SENTRY_ORG: redacted
      with:
        environment: ${{ inputs.env_name }}
        version: ${{ inputs.version }}
        sourcemaps: './public/js ./public/assets/js'
        strip_common_prefix: 'true'
        url_prefix: '~'
```

### Expected Result

Files would get uploaded stripping the common prefix (ie `public`), but instead get stripped way too much. If we look at the directory breakdown we have:

```
public/assets/js/*.chunk.js
public/js/*.js
```

### Actual Result

<img width="776" alt="Screenshot 2023-07-17 at 12 04 12 PM" src="https://github.com/getsentry/sentry-fastlane-plugin/assets/611784/d10deb97-f21c-4d60-a360-baf885d3ad21">

Basically we see it stripped way more than I expect. I would imagine the only common difference between those two above paths is up to the `public` part.

I expected to get files uploaded like:

 * `assets/js/*.chunk.js`
 * `js/*.js`
 
However, it stripped `assets/js` from the first and `js` from the 2nd. Which of course in my eyes is the reason the sourcemaps are not working
 
 > We've encountered 1 problem un-minifying your applications source code!

**Comment by philipphofmann** [comment]
This issue was opened at https://github.com/getsentry/sentry-fastlane-plugin, but I think it belongs here.

**Comment by iBotPeaches** [comment]
thanks.

I see this is probably because the loop is just making one-off calls to the CLI - https://github.com/getsentry/action-release/blob/master/src/main.ts#L48-L60C22

Maybe that parameter was intended to be used for all the files in one directory (to clean that path). When I passed two directories and assumed it would find the common of both - thats not how its working.

Since it'll just map/format each entry in my list independently. I'll do some research into what the CLI can take, since if it can eat multiple sourcemaps at once - this might resolve itself.

However,
* https://github.com/getsentry/action-release/issues/158 - seems like they are trying same issue - to align a prefix with a multiple set of sourcemaps (presuming the auto strip didn't work)
* https://github.com/getsentry/action-release/issues/116 - seems partially related since globbed directories as an input might then natively support the common path to strip from that result
* https://github.com/getsentry/action-release/issues/101  - its still on node12 and sentry-cli v1 so probably not going to gain much help from the CLI new features if its quite out of date.

**Comment by szokeasaurusrex** [comment]
> I see this is probably because the loop is just making one-off calls to the CLI

I believe the Sentry CLI should be able to take multiple sourcemap paths at once. So, fixing might just be a matter of passing all the paths to the Sentry CLI at the same time in the plugin
